src/TextArea/TextArea.less

```diff
@@ -19,34 +19,39 @@
   top: @size_xs;
   width: 100%;
   color: @neutral_dark_gray;
+}
 
-  .TextArea--label {
-    left: @size_s;
-    opacity: 1;
-  }
+.TextArea--label {
+  left: @size_s;
+  opacity: 1;
+  background-color: @neutral_white;
+  padding-right: @size_3xs;
+}
 
-  .TextArea--required {
-    position: absolute;
-    right: @size_l;
-  }
+.TextArea--required {
+  position: absolute;
+  right: @size_l;
+}
 
-  .TextArea--error {
-    color: @alert_red;
-    position: absolute;
-    right: @size_l;
-  }
+.TextArea--error {
+  color: @alert_red;
+  position: absolute;
+  right: @size_l;
 }
 
 .TextArea--input {
   .text--medium;
   display: block;
   width: 100%;
   .border--s(transparent);
+  border-left: @size_none;
```

Comment:
This left border removal plus the left padding change on [line 50](https://github.com/Clever/components/pull/123/files#diff-f33c3b7a08d6659c4f1c4293d40f34cbR50) make the left alignments of text in the `TextArea` and `TextInput` components match. These alignments used to be off by 2 pixels.